js/bybit.js

```diff
@@ -2190,16 +2226,20 @@ module.exports = class bybit extends Exchange {
             'info': response,
         };
         const data = this.safeValue (response, 'result', {});
-        const balances = this.safeValue (data, 'balances');
+        const balances = this.safeValue2 (data, 'balances', 'loanAccountList');
         if (Array.isArray (balances)) {
             // spot balances
             for (let i = 0; i < balances.length; i++) {
                 const balance = balances[i];
-                const currencyId = this.safeString (balance, 'coin');
+                const currencyId = this.safeString2 (balance, 'coin', 'tokenId');
                 const code = this.safeCurrencyCode (currencyId);
                 const account = this.account ();
-                account['free'] = this.safeString (balance, 'availableBalance');
-                account['used'] = this.safeString (balance, 'locked');
+                const locked = this.safeString (balance, 'locked');
+                const loan = this.safeString (balance, 'loan');
+                const interest = this.safeString (balance, 'interest');
+                const owed = Precise.stringAdd (loan, interest);
+                account['free'] = this.safeString2 (balance, 'availableBalance', 'free');
+                account['used'] = Precise.stringAdd (locked, owed);
```

Comment:
@samgermain would not be easier/efficient to let `safeBalance` compute `used` by doing `total - free = used`?

Reply:
That wouldn't be correct. 

- The total amount is the amount that the user owns.
- The free amount is the amount unused that is in the users wallet
- The used amount is the amount of currency that is locked up and not available for use, which includes the amount of borrowed currency and the amount of currency in trades

I'm realizing that this method might not be correct either. I'm not sure what the correct method would be. We should probably get the opinion of @kroitor @frosty00

Reply:
@samgermain any update?

Reply:
@carlosmiei @kroitor @frosty00 

No, I need a second opinion